libs/app-debug.js

```diff
@@ -0,0 +1,7477 @@
+/**
```

Comment:
Is it a good idea to commit `app-debug.js`? Or is this a file generated during development that could be excluded with `.gitignore`?

Reply:
Don't know whether it's a good or bad idea. Nevertheless, in the release package I include only compiled js files. Do you think it can decrease performance for someone who installs the theme directly from Git for production?

Reply:
Removed